integrations/operator/crdgen/testdata/protofiles/teleport/legacy/client/proto/event.proto

```diff
@@ -175,7 +178,11 @@ message Event {
     teleport.clusterconfig.v1.AccessGraphSettings AccessGraphSettings = 61;
     // SPIFFEFederation is a resource for SPIFFE federation.
     teleport.machineid.v1.SPIFFEFederation SPIFFEFederation = 62;
-    // StaticHostUser is a resource for static host users.
-    teleport.userprovisioning.v1.StaticHostUser StaticHostUser = 63;
+    // AutoUpdateConfig is a resource for autoupdate config.
```

Comment:
Why are the proto changes necessary? Are they part of the fix for k8s.io v1.31.x?

Reply:
Tiago re-syced the test snapshots fopr the CRD generation. We use a copy of the proto definitions rather than the real ones in the snapshot tests. It's dumb and not useful but I never got time to clean this up, so we need to sync the test inputs from time to time.

Reply:
This is fine, but I want to understand what is part of the k8s v1.31.x fix and what is just tagging along.

Reply:
The protobuf updates are unrelated but since I generated the new CRD stubs, I also updated the protos.